### PR TITLE
[IN-0] starting cmd-bridge server on base builders

### PR DESCRIPTION
**Pull request by fehersanyi-bitrise**
### New Pull Request Checklist

*Please mark the points which you did / accept.*

- [ ] The tool I added is stable, and __does not require frequent updates__. _Preinstalled tools on the LTS stacks
  are not updated, so if the tool requires frequent updates you should handle the installation on-demand (see: [Install Any Additional Tool - DevCenter](https://bitrise-io.github.io/devcenter/tips-and-tricks/install-additional-tools/) for more information)_
- [ ] I added a version report line to [`system_report.sh`](/system_report.sh) for the new tool(s) I added
- [ ] I have added an entry to the CHANGELOG.md with a revisionID (the current date and the number of the change e.g.: 2019_10_11_1) and a brief description of the actual change
- [ ] I have updated the revisionID in [bitrise_profile](roles/profiles/files/bitrise_profile) **BITRISE_OSX_STACK_REV_ID** with the one I just created in the CHANGELOG.md

### CODE REVIEW

**Comment by bitrisekristof** [comment]
Why dont we do it when we are creating the linkfroms?

File: roles/intermediate-setup/tasks/main.yml [hunk 1]
```diff
@@ -126,6 +126,13 @@
 - name: Sync the tools dir
   synchronize: src=tools/ dest="/Users/{{ param_user }}/bitrise/tools/"
 
+- name: make cmd-bridge executable
+  shell: bash -l -c "chmod +x bitrise/tools/cmd-bridge/_scripts/install_launchctl_plist_for_current_user.sh"
```
Comment (bitrise-tom) [inline roles/intermediate-setup/tasks/main.yml:130]: can we stop using shell commands and start using the built-on modules in ansible?

https://docs.ansible.com/ansible/latest/modules/file_module.html

Comment (fehersanyi-bitrise) [reply roles/intermediate-setup/tasks/main.yml]: been there, done that

File: roles/android-sdk-packages/tasks/main.yml [hunk 2]
```diff
@@ -59,9 +59,9 @@
 - name: Recursively change ownership of ANDROID_HOME
   become: yes
   file:
-    path: "{{ android_home }}"
-    state: directory
+    path: "/usr/local/Caskroom/android-sdk || /Users/vagrant/Library/Developer/Xamarin/android-sdk-macosx)"
```
Comment (bitrise-tom) [inline roles/android-sdk-packages/tasks/main.yml:62]: i'm not sure if this || will work here? i needs splitting into 2

Comment (fehersanyi-bitrise) [reply roles/android-sdk-packages/tasks/main.yml:62]: the test was successful 🧐 I could rerun it on a vs4mac stack to see if it finds it on that too, but it didn't throw an error on edge

Comment (fehersanyi-bitrise) [reply roles/android-sdk-packages/tasks/main.yml:62]: oh actually this run on a vs4mac stack and it found the right path 🎉

Comment (fehersanyi-bitrise) [reply roles/android-sdk-packages/tasks/main.yml:62]: @bitrise-tom 👆

Comment (bitrise-tom) [reply roles/android-sdk-packages/tasks/main.yml:62]: But, there is a trailing `)` on the second path, it isn't right

Comment (fehersanyi-bitrise) [reply roles/android-sdk-packages/tasks/main.yml:62]: oh god okay

**Pull request merged**